### DISCUSSION TITLE
bypass Malwarebytes Firewall

### DISCUSSION THREAD

**Discussion by kiro5678** [HotCakeX/Harden-Windows-Security#369] [Q&A]
Enabling Microsoft Security Baselines causes applications to bypass Malwarebytes Firewall What is the solution?

**Comment by HotCakeX** [comment]
I think you're referring to Windows Firewall control that is apparently a program saying it's from Malwarebytes but hosted on a different website, looks like it has its own configurations to manage Windows Firewall. Maybe they have to update it to be compatible with new version of Windows or it has an issue with a policy configured by the security baselines. They should apply the Microsoft security baselines on a system and test their software in there to debug it

**Comment by dannygee87** [comment]
Hi,

I had the same problem. I found the following solution, although I'm still testing it and not entirely sure it's the solution.

The problem, that WFC no longer asks what it should allow, is that the baselines set the default firewall policies to block inbound and allow outbound if no rule matches. Since outbound connections are then allowed anyway, WFC essentially does nothing, because its allowed.

To change this, you need to open the Group Policy Editor via gpedit.msc and navigate to "Computer Configuration > Windows > Settings > Security > Windows Defender... "(sorry, not sure how it's translated into English).

Then right-click and select Attributes. Here you can set the default policy for inbound and outbound connections if no rule matches.
Select the profile you are using and set the default outgoing to block. Then WFC will ask again whether it should allow connections, if not already a rule matches.
